tests/dummy/app/styles/app.scss

```diff
@@ -1,10 +1,10 @@
-@import 'variables/variables';
+@use 'variables/_variables' as *;
```

Comment:
[question] I don't _think_ the underscore is needed anymore (to have Sass ignore certain files). Is it some form of convention used locally? (I see the other files are "imported/used" without the underscore.